.gitignore

```diff
@@ -1,4 +1,6 @@
 *.sublime-*
 
+.idea
```

Comment:
Can you configure this in your [global .gitignore](https://help.github.com/articles/ignoring-files/)?